package.json

```diff
@@ -10,7 +10,7 @@
     "dev": "concurrently \"npm run server:dev\" \"npm run client:dev\" \"npm run storybook\"",
     "serve": "vite preview",
     "prepare": "husky install",
-    "test": "prettier --check . && eslint . && stylelint \"**/*.css\"",
+    "test": "prettier --check . && eslint .",
```

Comment:
There's also a way to enable stylelint for styled components, but it's not as comfortable, because you need to exec a command to display errors etc. 
https://styled-components.com/docs/tooling#stylelint